target/linux/mvebu/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 BOARD:=mvebu
 BOARDNAME:=Marvell EBU Armada
 FEATURES:=fpu usb pci pcie gpio nand squashfs ramdisk boot-part rootfs-part
-SUBTARGETS:=cortexa9 cortexa53 cortexa72
+SUBTARGETS:=cortexa9 cortexa9-neon cortexa53 cortexa72
```

Comment:
Note that there are only two non-alphanumeric subtargets in entire trunk (lantiq/xway_legacy and layerscape/armv8_64b) which both use underscore. While I actually prefer the hyphen (just matter of taste), please make sure that having a hyphen in subtarget name doesn't break anything (or use underscore instead).